packages/docs-page/server/loaders/remote-content.ts

```diff
@@ -125,6 +132,9 @@ export default class RemoteContentLoader implements DataLoader {
           '`remarkPlugins:` When specified as a function, must return an array of remark plugins'
         )
       }
+    } else {
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion -- we default this in the constructor, so it must be defined
+      remarkPlugins = this.opts.remarkPlugins!
```

Comment:
Not for this PR, but it might be nice refactor `opts` into separate class properties so we don't need to do stuff like this.